packages/usa-input-prefix-suffix/src/styles/_usa-input-prefix-suffix.scss

```diff
@@ -58,6 +58,13 @@ $icon-offset: $icon-width + ($input-padding * 2);
   }
 }
 
+// Create input group max width utility classes
+@each $name, $width in $system-input-widths {
+  .usa-input-group--#{$name} {
+    max-width: #{$width};
+  }
+}
+
```

Comment:
**thought:** These classes feel a bit redundant.

I understand why we might not want to drop support for them but it feels like we could set these styles with `usa-input--` classes or even better, the `maxw-` utility classes. 

Since we're addressing them now, I wanted to bring this up in case we wanted to explore other options for how users would customize these inputs. That said, we might be better for simply supporting them because users might already be using these classes. Let me know what you think!

Reply:
I originally thought the same thing, but ended up landing on keeping them for the following reasons:
1. It felt more more intuitive to add `usa-input-group--[width]` to `usa-input-group` elements and `usa-input--[width]` classes to `usa-input` elements. It felt weird to mix them. 
2. The `maxw` classes are not ideal for this since they are defined with `rem` rather than `ex`. The `ex` unit makes more sense for inputs, and having a set of standard widths for inputs makes sense to me. 
3. And most importantly, preserving the classes as they are keeps this from being a breaking change. 
 
How we set input widths is something worth exploring in the Elements version of the component, though. Good to flag it for future discussion.